### PR TITLE
Gorlex Girlfriends Mail

### PR DESCRIPTION
**Pull request by Unkn0wnGh0st333**
## About the PR
Adds a new mail type that includes a note and a copy of this month's Gorlex Girlfriends Magazine. This edition includes a behind-the-scenes look at the "Nukies" TV show!

## Why / Balance
This is necessary,

## Technical details
New item - Gorlex Magazine
New Note - Subscription Notice
New Mail Type - Gorlex Girlfriend mail at 0.5 chance

## Media
image

## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
Shakes the very fabric of reality

**Changelog**
🆑Unkn0wn_Gh0st, K8art
- add: A recent rise of Gorlex Girlfriend magazines have been seen on Nanotrasen stations and arriving in the mail... Odd...

### CODE REVIEW

**Comment by Kr8art** [comment]
> New Mail Type - Gorlex Girlfriend male at 0.5 chance

Ah yes, the elusive male girlfriend.

**Comment by Unkn0wnGh0st333** [comment]
> > New Mail Type - Gorlex Girlfriend male at 0.5 chance
> 
> Ah yes, the elusive male girlfriend.

Killing you

File: Resources/Locale/en-US/deltav/paper/paper-misc.ftl [hunk 1]
```diff
@@ -9,4 +9,13 @@ book-text-lunchbox-healthy = Hello Honey!
...
       I've packed you something fun for this shift! I hope you enjoy it!!
       Love you so so much,
-       - Mum.
\ No newline at end of file
+       - Mum.
+book-text-subscription-notice = [head=2]Greetings loyal subscriber,[/head]
+      
+      We hope you enjoy this months edition of Gorlex Girlfriends™ sponsored by the Syndicate.
+      
+      With this months issue we will be heading behind the seens with some of our famous actors working on the amazing "[color=#780606]Nukies[/color]" TV show and maybe show some teasers for next season. Read up on the personal interview with beloved Operative November as she explains the troubles of fake blood getting in her hair!
```
Comment (deltanedas) [inline Resources/Locale/en-US/deltav/paper/paper-misc.ftl]: scenes not seens

File: Resources/Locale/en-US/deltav/paper/book-salvage.ftl [hunk 2]
```diff
@@ -50,3 +50,9 @@ book-text-vulpkanin = Vulpkanin for Dummies, an Exhaustive Guide
        How to Kiss: Kiss
 
        Don't Underestimate Them: Vulpkanin may look cute and cuddly, but they can be fierce fighters when provoked. Treat them with respect and avoid unnecessary conflict.
+
+book-text-gorlexgirlfriend = It's the 197 version of the infamous Gorlex Girlfriends magazine. This issue includes several pictures and interviews of multiple agents working in the syndicate alongside actors who work on the Syndicate TV show "Nukies". The best part by far was about Operative November who is the star of this edition, with their picture being on the cover.
```
Comment (deltanedas) [inline Resources/Locale/en-US/deltav/paper/book-salvage.ftl]: 197 version -> 197th issue
their picture -> her picture (same pronouns as below)

File: Resources/Prototypes/DeltaV/Entities/Objects/Misc/books.yml [hunk 3]
```diff
@@ -44,3 +44,43 @@
       color: "#2c5491"
   - type: Paper
     content: book-text-vulpkanin
+
+- type: entity
+  parent: BaseItem
+  id: BookGorlexGirlfriends
+  name: Gorlex Girlfriends - Issue 197
+  description:
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Objects/Misc/books.yml]: remove or add a description

File: Resources/Textures/DeltaV/Objects/Misc/gorlex_magazine.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Objects/Misc/gorlex_magazine.rsi/meta.json]: 4 space indent for meta.json

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Locale/en-US/deltav/paper/paper-misc.ftl (hunk 1 above)
Comment (Unkn0wnGh0st333) [reply Resources/Locale/en-US/deltav/paper/paper-misc.ftl]: Grammarly really didn't help me much with helping these fuckups smh

**Review by deltanedas — APPROVED** [review]

**Review by Lyndomen — APPROVED** [review]

**Pull request merged**